From c167f35fbb76c4246c730b29262a59da73010412 Mon Sep 17 00:00:00 2001
From: Alex Tsitsimpis <alex.tsitsimpis@gmail.com>
Date: Thu, 4 Jun 2020 12:49:40 +0300
Subject: [PATCH] Fix build with Qt 5.15+
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Building with Qt 5.15 failed with the following error:

../widgets/pixmapdial.cpp:231:26: error: aggregate ‘QPainterPath ballPath’ has incomplete type and cannot be defined
  231 |             QPainterPath ballPath;
      |                          ^~~~~~~~
../widgets/pixmapdial.cpp:278:26: error: aggregate ‘QPainterPath ballPath’ has incomplete type and cannot be defined
  278 |             QPainterPath ballPath;
      |                          ^~~~~~~~

Signed-off-by: Alex Tsitsimpis <alex.tsitsimpis@gmail.com>
---
 c++/widgets/pixmapdial.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/c++/widgets/pixmapdial.cpp b/c++/widgets/pixmapdial.cpp
index 37dcffd..0b5e0f5 100644
--- a/c++/widgets/pixmapdial.cpp
+++ b/c++/widgets/pixmapdial.cpp
@@ -22,6 +22,7 @@
 #include <QtCore/QTimer>
 #include <QtGui/QPainter>
 #include <QtGui/QPaintEvent>
+#include <QtGui/QPainterPath>
 
 PixmapDial::PixmapDial(QWidget* parent)
     : QDial(parent),
